ampd/src/stellar/verifier.rs

```diff
@@ -319,18 +314,16 @@ mod test {
             topics: vec![
                 ScVal::Symbol(ScSymbol(StringM::from_str(TOPIC_CALLED).unwrap())),
                 ScVal::Address(msg.source_address.clone()),
+                ScVal::String(msg.destination_chain.clone()),
+                ScVal::String(msg.destination_address.clone()),
                 ScVal::Bytes(msg.payload_hash.clone()),
             ]
             .try_into()
             .unwrap(),
             data: ScVal::Vec(Some(
```

Comment:
Event data is not a tuple containing payload anymore, but the payload itself `ScVal::Bytes(..)`

Reply:
just change this to non vec
```

Reply:
updated.